packaging/neo4j-desktop/src/main/java/org/neo4j/desktop/config/unix/UnixInstallation.java

```diff
@@ -48,9 +48,9 @@ public File getVmOptionsFile()
     }
 
     @Override
-    public File getServerConfigurationsFile()
+    public File getConfigurationsFile()
     {
-        return new File( getConfigurationDirectory(), "." + NEO4J_SERVER_PROPERTIES_FILENAME );
+        return new File( getConfigurationDirectory(), "." + NEO4J_PROPERTIES_FILENAME );
```

Comment:
I know there's been trickery around config files in Desktop before (and literally, that is all I know about it) - do we know that this `NEO4J_PROPERTIES_FILENAME` file is in an editable location that users can find?